cimgen/languages/modernpython/utils/export_template.mustache

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
i can't comment much for other languages, but in the case of "modern" python, i believe that reflection is a better and cleaner approach to generate the rdf out of a collection of cim class instances than templating.

Reply:
The writer is inspired by the writer of cimpy. I added this because I couldn't find a writer for modernpython. Did I miss something?